app/src/main/java/com/swent/assos/config/Config.kt

```diff
@@ -1,10 +1,74 @@
 package com.swent.assos.config
 
+import com.swent.assos.R
+import java.net.HttpURLConnection
+import java.net.URL
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import org.json.JSONObject
+
 class Config {
+  data class Service(val name: String, val host: String, val port: Int?)
+
+  suspend fun checkFirebaseEmulator(ip: String, port: String): Boolean {
+    val url = URL("http://${ip}:${port}")
+    val connection = url.openConnection() as HttpURLConnection
+    connection.requestMethod = "GET"
+
+    var connected = false
 
-  val demo = false
+    return withContext(Dispatchers.IO) {
```

Comment:
avoid hardcoded dispatchers, I managed to inject a dispatcher in another class but we should discuss about wether it is really needed or not